graql/language/update.feature

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright (C) 2021 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+#noinspection CucumberUndefinedStep
+Feature: Graql Delete Query
+
+  Background: Open connection and create a simple extensible schema
+    Given connection has been opened
+    Given connection does not have any database
+    Given connection create database: grakn
+    Given connection open schema session for database: grakn
+    Given session opens transaction of type: write
+
+    Given graql define
+      """
+      define
+      person sub entity,
+        plays friendship:friend,
+        plays parenthood:parent,
+        plays parenthood:child,
+        owns name,
+        owns ref @key;
+      friendship sub relation,
+        relates friend,
+        owns ref @key;
+      parenthood sub relation,
+        relates parent,
+        relates child;
+      name sub attribute, value string;
+      ref sub attribute, value long;
+      """
+    Given transaction commits
+
+    Given connection close all sessions
+    Given connection open data session for database: grakn
+    Given session opens transaction of type: write
+
+
+  ##########
+  # THINGS #
+  ##########
```

Comment:
If this comment for consistency with elsewhere? If not it's meaningless as we're always talking things.